data-avro/src/test/java/com/linkedin/data/avro/TestSchemaTranslator.java

```diff
@@ -1660,6 +1660,46 @@ public void testToAvroSchema_testTypeRefAnnotationPropagation(String schemaBefor
               null,
               null,
               null
+          },
+          {
+              " { " +
+              "   \"type\" : \"record\", " +
+              "   \"name\" : \"Foo\", " +
+              "   \"fields\" : [ { " +
+              "     \"name\" : \"field1\", " +
+              "     \"type\" : \"int\", " +
+              "     \"b_customerAnnotation\" : \"f1\", " +
+              "     \"c_customerAnnotation\" : \"f1\", " +
+              "     \"a_customerAnnotation\" : \"f1\" " +
```

Comment:
nit: I think you mean to say `x_customAnnotation`?

Reply:
yes sir.